client/lib/post-normalizer/utils.js

```diff
@@ -30,6 +30,16 @@ export function imageSizeFromAttachments( post, imageUrl ) {
 	}
 }
 
+export function stripPhotonParams( imageUrl ) {
+	if ( ! imageUrl ) {
+		return imageUrl;
+	}
+	const parsedUrl = url.parse( imageUrl );
+	parsedUrl.query = parsedUrl.search = ''; // remove query params
```

Comment:
I don't know enough about how various themes modify Photon params.
My gut says we shouldn't keep anything but I'll defer to @blowery's thoughts here